src/Symfony/Component/Serializer/Tests/Normalizer/BackedEnumNormalizerTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\Normalizer;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+use Symfony\Component\Serializer\Normalizer\BackedEnumNormalizer;
+use Symfony\Component\Serializer\Tests\Fixtures\IntegerBackedEnumDummy;
+use Symfony\Component\Serializer\Tests\Fixtures\StringBackedEnumDummy;
+use Symfony\Component\Serializer\Tests\Fixtures\UnitEnumDummy;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+class BackedEnumNormalizerTest extends TestCase
+{
+    /**
+     * @var BackedEnumNormalizer
+     */
+    private $normalizer;
+
+    protected function setUp(): void
+    {
+        $this->normalizer = new BackedEnumNormalizer();
+    }
+
+    /**
+     * @requires PHP 8.1
+     */
```

Comment:
moving the annotation on the class would avoid duplicating it on each test

Reply:
There are also tests for PHP 7 in this file.

Reply:
indeed, I missed the `<`.